recipes/openmm-cpu/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "openmm" %}
+{% set version = "7.4.0" %}
+{% set commit = "b71e92d9ccef7c98cfd6b862eb34095d94fa1b05" %}
+{% set build = 0 %}
+
+package:
+  name: openmm-cpu
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/{{ name }}/archive/{{ commit }}.tar.gz
+  sha256: 720c9aa0215e2dd851c40a6319814869cb55aa82ef908476c76bec3d494dec1d
+
+
+build:
+  number: {{ build }}
+  string: "rc_py{{ py }}_h{{ PKG_HASH }}_{{ build }}"
```

Comment:
Why is this string needed?

Reply:
I am adapting the original [`omnia` recipe](https://github.com/omnia-md/conda-recipes-cuda/blob/master/openmm/meta.yaml#L22), and also read about it in the [docs](https://conda-forge.org/docs/maintainer/knowledge_base.html#preferring-a-provider-usually-nompi). Since it was being used to specify the CUDA version in the [sibling recipe](https://github.com/conda-forge/staged-recipes/pull/9102), it ended up looking like this. Of course, please suggest the best practices here. @jchodera might want to add something too.

Reply:
I don't think we need a build string here, and possibly not even in the CUDA recipe either. 

Previously, on the `omnia` build infrastructure, we were building different variants for each CUDA Toolkit version. For conda-forge, for GPU-enabled OpenMM, we can deliver a specific `cudatoolkit`as a dependency and make sure OpenMM gets compiled against that version, so we may not need build strings there either.

Reply:
Removing the `string` line for now.